radixdlt-java/src/main/java/com/radixdlt/client/atommodel/timestamp/TimestampParticle.java

```diff
@@ -26,7 +26,6 @@ public TimestampParticle(long timestamp) {
 	}
 
 	public Long getTimestamp() {
```

Comment:
Appreciate this didn't change, but I think this should be a `long`, not a wrapped `Long`.